content/help/sso/social-connect.md

```diff
@@ -20,7 +20,7 @@ aliases:
 - /help/integrations/github
 - /help/addons/social/github
 - /help/addons/social/facebook
-- /help/addons/social/googleplus
```

Comment:
Don't do this. These are redirects from old pages. You're essentially removing an old redirect, and adding a redirect from a page that never existed.